src/Vehicle/Vehicle.h

```diff
@@ -986,14 +994,17 @@ private slots:
 
     static void _rebootCommandResultHandler(void* resultHandlerData, int compId, const mavlink_command_ack_t& ack, MavCmdResultFailureCode_t failureCode);
 
+    // send mavlink command (deprecated since Aug 2022)
+    Q_INVOKABLE void setCurrentMissionSequenceWithMissionSetCurrent(int seq);
```

Comment:
I would make this non-invokable and private. I don't think we want anyone using this except for the lambda fallback case. I might put lambdaFallback in the name to steer people away from using it in other Vehicle code.

Reply:
Does that make sense?

Reply:
I've pushed up a patch on top which simply inlines the function into the lambda itself.  I was avoiding that to reduce the patch size - but it does neaten things up a bit.

Thoughts?

Reply:
Yeah, thanks Peter. I like that much better.